dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -303,3 +412,26 @@ void CommandBarFlyout::SetSecondaryCommandsToCloseWhenExecuted()
         }
     }
 }
+
+void CommandBarFlyout::AddDropShadow()
+{
+    if (SharedHelpers::Is21H1OrHigher())
+    {
+        if (m_presenter)
```

Comment:
>m_presenter [](start = 12, length = 11)

nit: the if Check should be `if(auto&& presetner = m_presenter.get())` because when a tracker ref is implicitly cast to a boolean it calls the get() method, so this prevents us from calling the getter twice.